src/components/Header/Header.tsx

```diff
@@ -1,14 +1,27 @@
 import React from "react";
 import PetsIcon from "@mui/icons-material/Pets";
+import { makeStyles } from "@mui/styles";
+import { Theme } from "@mui/material";
+
+const useStyles = makeStyles((theme: Theme) => ({
+  SpalshDiv: {
```

Comment:
@ZainShahbaz please use camelCase for variables and almost everything
```suggestion
  spalshDiv: {
```
Examples are here
https://www.w3schools.com/js/js_conventions.asp